components/contextMenu/contextMenu.tsx

```diff
@@ -261,15 +292,17 @@ export default function CustomContextMenu(props: Props) {
                           items={snoozeItem}
                           iconSize='12'
                           iconViewBox='0 0 12 12'
-                          onClose={() => setShowDropDown(false)}
+                          onClose={() => {}}
```

Comment:
Remove this onClose props if it's not required

Reply:
Done